### PR TITLE
 Peformed every css project successfully and made the websites complete responsive

### PR DESCRIPTION
**Pull request by ankit071105**
# Peformed every css project successfully and made the websites complete responsive

Screenshot 2023-10-02 at 1 01 59 PM

Screenshot 2023-10-02 at 1 21 26 PM

Screenshot 2023-10-02 at 1 05 15 PM

Screenshot 2023-10-02 at 1 25 25 PM
ddc)

### CODE REVIEW

**Comment by stevemao** [comment]
thank you @ankit071105 could you please resolve the conflict?

**Comment by ankit071105** [comment]
@stevemao 
✔️ Done

File: examples/menu-1.html [hunk 1]
```diff
@@ -10,6 +10,100 @@
...
+        }
+        @media (min-width: 800px){
+            body{
+                width: 90%;
+            }
+            .menu{
+                width: 70%;
+            }
+        }
+=======
```
Comment (stevemao) [inline examples/menu-1.html:106]:
```suggestion

```

File: examples/menu-1.html [hunk 2]
```diff
@@ -190,14 +284,58 @@
...
+                                        <a href="">mentok satu</a>
+                                    </li>
+                                </ul>
+                            </li>
+                            <li>
+                                <a href="">jeruh satu</a>
+                            </li>
+                        </ul>
+                    </li>
+=======
```
Comment (stevemao) [inline examples/menu-1.html:338]:
```suggestion

```

**Pull request merged**